client/ayon_core/hosts/houdini/plugins/publish/validate_cop_output_node.py

```diff
@@ -71,8 +58,9 @@ def get_invalid(cls, instance):
         # the isinstance check above should be stricter than this category
         if output_node.type().category().name() != "Cop2":
             raise PublishValidationError(
-                (
-                    "Output node {} is not of category Cop2."
-                    " This is a bug..."
-                ).format(output_node.path()),
-                title=cls.label)
+                f"Output node {output_node.path()} is not of category Cop2.",
+                description=(
+                    "### Invalid COP output node\n\n"
+                    "The output node path for the instance must be set to a "
+                    "valid COP node path. See the log for more details."
+                ))
```

Comment:
for sake of consistency, 
should this be: 
```suggestion
            cls.log.error(
                f"Output node {output_node.path()} is not of category Cop2.",
            )
            return [output_node.path()]
```